packages/firestore/package.json

```diff
@@ -46,7 +44,8 @@
     "test:node:persistence:prod": "node ./scripts/run-tests.js --main=index.node.ts --persistence 'test/{,!(browser|lite)/**/}*.test.ts'",
     "test:travis": "ts-node --compiler-options='{\"module\":\"commonjs\"}' ../../scripts/emulator-testing/firestore-test-runner.ts",
     "test:minified": "(cd ../../integration/firestore ; yarn test)",
-    "api-report": "rm -rf temp && api-extractor run --local --verbose",
+    "prepare": "yarn build:release",
```

Comment:
```suggestion
```
prepare scripts were removed in https://github.com/firebase/firebase-js-sdk/pull/4296 to speed up releases.